src/use_case/search_nearby/SearchNearbyInteractor.java

```diff
@@ -0,0 +1,26 @@
+package use_case.search_nearby;
+
+import entity.Events.Event;
+
+import java.sql.Array;
+import java.util.ArrayList;
+
+public class SearchNearbyInteractor implements SearchNearbyInputBoundary{
+    SearchNearbyDataAccessInterface dataAccessObject;
```

Comment:
I suggest that you change the name of the DAO to be more specific on what type of DAO it is.